etna/lib/etna/route.rb

```diff
@@ -176,13 +177,21 @@ def update_params(request)
       )
     end
 
+    def separator_free_match
```

Comment:
I noticed an issue with the way we handle named args; when I want to serve workflows or data files from a named argument, it considers `.` a separator, which may make sense in some cases, but in our data and cwl endpoints, it does not.  So, I added an option to enable matching named args at the extension level, too.

Reply:
Err, missed this the other day but just noticed it - is this necessary? Can't you use the `*` to catch these names? E.g. in vulcan:  `get 'api/:project_name/data/:cell_hash/*data_filename', action: 'data#fetch', as: :data_view, auth: { user: { can_view?: :project_name } }`. I guess it couldn't handle two globs like `*cell_hash/*data_filename` but otherwise it ought to work.